.github/workflows/README.md

```diff
@@ -0,0 +1,126 @@
+# AlphaX Release Process
+
+## Release a new version
+### Tagging of changes
+In order to have release notes automatically generated, changes need to be tagged with labels.
```

Comment:
which exact changes? Could we formulate it somewhat like: if the user wants do develop in their own repo with compatibility to alphashared, they need to adapt the following tag schema when they make changes on the level XX (PR/commit or so). Could it make sense to move the "tagging of changes" section below the "release a new version" section?

Reply:
I removed this altogether, it was confusing to mix this in.
This tagging scheme is to automatically fill release notes, which has nothing to do with this workflow, and, on top, is the responsibility of each separate repo. (cf. https://github.com/MannLabs/alphadia/blob/main/.github/release.yml)